tests/python/gaia-ui-tests/gaiatest/tests/functional/rocketbar/test_rocketbar_offline_behavior.py

```diff
@@ -30,7 +30,7 @@ def test_rocketbar_offline_behavior(self):
 
         settings = search_panel.tap_offline_settings_button()
 
-        settings.wait_for_airplane_toggle_ready()
+        settings.wait_for_airplane_mode_ready()
         self.assertTrue(settings.is_cell_data_menu_visible)
         self.assertTrue(settings.is_wifi_menu_visible)
-        self.assertTrue(settings.is_airplane_mode_visible)
+        self.assertTrue(settings.is_airplane_mode_displayed)
```

Comment:
I wonder what are these assertions for. Why do we want to make sure these entries are visible? @npark-mozilla @mwargers Do you guys see any value in them?